composables/useListInfiniteScroll.ts

```diff
@@ -153,7 +159,6 @@ export default function ({
     await fetchDataCallback(nextPage, 'down', () => {
       endPage.value = nextPage
       checkAfterFetchDataSuccess()
-      prefetchNextPage()
```

Comment:
I see you have added `distance: 1000,` to do the prefetch. However, removing this function would make the process of loading the next page slower, compared to the canary env.

```
https://canary.kodadot.xyz/ksm/explore/items
```

How about reducing the times of prefetch?

Reply:
updated, readd back this line